projects/3-way-negotiation/instructions_survey.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
There are a couple files here that are probably leftover from a copy/paste that you don't need anymore - can you pull them out just for tidyness?